### PR TITLE
LWIMIQ embedding ansible additions edition

### PR DESCRIPTION
**Pull request by jjlangholtz**
Another LWIMIQ post!

This is the first blog post I have written in a long while so I would appreciate any and all feedback 😄

@JPrause @syncrou @imtayadeway

### CODE REVIEW

**Comment by JPrause** [comment]
LGTM! Nice job @jjlangholtz

**Comment by JPrause** [comment]
@jrafanie could you also review.

File: site/_posts/2017-02-27-embedding-ansible-additions.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+Another outstanding week with over 109 PRs merged. Here is the breakdown:
+
+* [49 from ManageIQ core][manageiq PRs merged]
+* [42 from the Classic UI][manageiq-ui-classic PRs merged]
+* [13 from the Service UI][manageiq-ui-service PRs merged]
+* [5 from the Amazon Provider][manageiq-providers-amazon PRs merged]
+
+## Improved
+
+There were quite a few improvements to facilitate the embedding Ansible effort. First, [Ladas](https://github.com/Ladas) has laid the foundation for [query optimizations](https://github.com/ManageIQ/manageiq/pull/13978) for inventory collections based on tracking graph dependencies. [Nick](https://github.com/carbonin) implemented a [worker lifecycle for embedded Ansible](https://github.com/ManageIQ/manageiq/pull/14023) to aid monitoring efforts. To simplify decorating models, [Martin](https://github.com/himdel) has added an implementation for [class decorator support](https://github.com/ManageIQ/manageiq-ui-classic/pull/237) which also removes the need to use the draper gem.
```
Comment (jrafanie) [inline site/_posts/2017-02-27-embedding-ansible-additions.md]: I think 13978 (worker lifecyle for embedded Ansible) and 14023 (query optimzations) links were swapped

Comment (jrafanie) [inline site/_posts/2017-02-27-embedding-ansible-additions.md]: I don't believe these are all related to the Ansible effort described in the first sentence, are they?  The first sentence makes it seem like the sentences following it are for PRs in the Ansible area.  I could be wrong but I don't think the decorators are.. maybe not even the query optimizations?

File: site/_posts/2017-02-27-embedding-ansible-additions.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+## Deleted
+
+Lots of deletions across the board over the last week. The classic UI received a pair of substantial removals: [Dávid](https://github.com/skateman) dropped the [unused jQplot and associated plugins](https://github.com/ManageIQ/manageiq-ui-classic/pull/445), while  [Harpreet](https://github.com/h-kataria) removed [arbitration profiles](https://github.com/ManageIQ/manageiq-ui-classic/pull/426).
+
+With as many jobs as we have running on TravisCI, it's nice when we can cut down on what we are checking. [Joe](https://github.com/jrafanie) dropped [the Codeclimate eslint engine](https://github.com/ManageIQ/manageiq/pull/14056) now that the UI is separate from the core. Finally [Nick](https://github.com/carbonin) took the opportunity to remove [unnecessary authentication configuration](https://github.com/ManageIQ/manageiq/pull/13966) with all the recent Ansible changes.
```
Comment (jrafanie) [inline site/_posts/2017-02-27-embedding-ansible-additions.md]: It's nice to see so many deletions highlighted!   ✂️ 🔥

**Comment by jrafanie** [comment]
@jjlangholtz  I had a few comments above.  Nice work!

File: site/_posts/2017-02-27-embedding-ansible-additions.md (hunk 1 above)
Comment (jjlangholtz) [inline site/_posts/2017-02-27-embedding-ansible-additions.md]: Thanks for catching the swapped links!

I don't believe they are all directly related to the Ansible effort either. I will update the wording for the intro sentence to something more accurate.

**Comment by JPrause** [comment]
@jjlangholtz did you have all the edits made. If yes, we'll see about getting this merged.

**Comment by jjlangholtz** [comment]
@JPrause Yes, I have addressed the feedback. 👍

**Comment by JPrause** [comment]
Cool! Thanks again @jjlangholtz for your willingness to help out.
@cybette or @hayesr when you have a minute, can you merge this blog entry.

**Pull request merged**